extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -1638,18 +1655,18 @@ static public function clearCache( $city_id ) {
 		/**
 		 * clear domains cache
 		 */
-		self::clearDomainCache( $city_id );
+		static::clearDomainCache( $city_id );
 
 		/**
 		 * clear variables cache
 		 */
 		$wgMemc->delete( "WikiFactory::getCategory:" . $city_id ); //ugly cat clearing (fb#9937)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)